### PR TITLE
[W5.3b][W14-A2]Ang Yee Chin 

### PR DESCRIPTION
**Pull request by Ang-YC**

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:3]: This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    public String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:4]: Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public int hashCode() {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:21]: In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!

**Review by okkhoy — COMMENTED** [review]
You can try to move the validation logic as well to the superclass.

**Comment by okkhoy** [comment]
@Ang-YC 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 3 above)
Comment (Ang-YC) [reply src/seedu/addressbook/data/person/Contact.java:21]: I think I left it by accident, I understand it is important when we need it in HashMap etc

File: src/seedu/addressbook/data/person/Contact.java (hunk 2 above)
Comment (Ang-YC) [reply src/seedu/addressbook/data/person/Contact.java:4]: Noted, will ensure some members are encapsulated next time

**Comment by Ang-YC** [comment]
Noted with thanks =)

**Pull request closed without merging**